tools/check_format.py

```diff
@@ -341,7 +341,7 @@ def checkSourceLine(line, file_path, reportError):
   if not whitelistedForGetTime(file_path):
     if "std::get_time" in line:
       if "test/" in file_path:
-        reportError("Don't use std::get_time; use TestUtility::parseTimestamp in tests")
+        reportError("Don't use std::get_time; use TestUtility::parseTime in tests")
```

Comment:
should we also look for std::mktime and exclude it?

Reply:
Yes, but it's still used in AccessLogFormatterTests: https://github.com/envoyproxy/envoy/blob/13f2665ed0bdd2901eb071e2ff38b53fa5c0bbea/test/common/access_log/access_log_formatter_test.cc#L539
and there is something funky going on there with timezones, but I don't have more time to dig into this right now, sorry.

I've opened an issue to track this, though: #5587